src/transactions/feed/detailContent/EarnContent.tsx

```diff
@@ -0,0 +1,171 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import RowDivider from 'src/components/RowDivider'
+import TokenDisplay from 'src/components/TokenDisplay'
+import { getDynamicConfigParams } from 'src/statsig'
+import { DynamicConfigs } from 'src/statsig/constants'
+import { StatsigDynamicConfigs } from 'src/statsig/types'
+import Colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { useTokenInfo } from 'src/tokens/hooks'
+import NetworkFeeRowItem from 'src/transactions/feed/detailContent/NetworkFeeRowItem'
+import { EarnClaimReward, EarnDeposit, EarnWithdraw } from 'src/transactions/types'
+
+interface EarnClaimRewardProps {
+  transaction: EarnClaimReward
+}
+
+export function EarnClaimContent({ transaction }: EarnClaimRewardProps) {
+  const { t } = useTranslation()
+  const { providerName } = getDynamicConfigParams(
+    DynamicConfigs[StatsigDynamicConfigs.EARN_STABLECOIN_CONFIG]
+  )
+  const tokenInfo = useTokenInfo(transaction.amount.tokenId)
+  const tokenSymbol = tokenInfo?.symbol ?? ''
+
+  return (
+    <>
```

Comment:
is there a reason the other ones are wrapped in Views, but this one isn't?

Reply:
Fixed in e36c3524b823607d4529693adb4e2e917ba38f69!